src/Umbraco.Web.Website/Controllers/UmbLoginStatusController.cs

```diff
@@ -56,4 +58,16 @@ public async Task<IActionResult> HandleLogout([Bind(Prefix = "logoutModel")] Pos
         // Redirect to current page by default.
         return RedirectToCurrentUmbracoPage();
     }
+
+    /// <summary>
+    ///     We pass in values via encrypted route values so they cannot be tampered with and merge them into the model for use
+    /// </summary>
+    /// <param name="model"></param>
+    private void MergeRouteValuesToModel(PostRedirectModel model)
+    {
+        if (RouteData.Values.TryGetValue(nameof(PostRedirectModel.RedirectUrl), out var redirectUrl) && redirectUrl != null)
```

Comment:
```suggestion
        if (RouteData.Values.TryGetValue(nameof(PostRedirectModel.RedirectUrl), out var redirectUrl) && redirectUrl is not null)
```

Reply:
Pattern matching FTW! The code was copied from one of the other controllers that handles this in a similar fashion... 😆